rpc/blockchain.go

```diff
@@ -188,7 +188,6 @@ func (s *PublicBlockchainService) GetBlockByNumber(
 			return nil, err
 		}
 	}
-	blockArgs.InclTx = true
```

Comment:
was this a bug that all query got txn included in the return?

Reply:
This looks like a deprecated field and it is not used in any cases. Thus I removed this field.